app/services/moderator/delete_user.rb

```diff
@@ -1,47 +1,15 @@
 module Moderator
   class DeleteUser < ManageActivityAndRoles
-    attr_reader :user, :admin, :user_params
+    attr_reader :user, :user_params
```

Comment:
So we don't need `admin` here anymore as a variable. I think this is fine since we don't add a `note` whenever to delete an account, unlike the other `Moderator` module actions.